docarray/base_document/document.py

```diff
@@ -43,3 +43,6 @@ def __str__(self):
             _console.print(self)
 
         return capture.get().strip()
+
+    def get_string_for_regex_filter(self):
```

Comment:
```suggestion
    def _get_string_for_regex_filter(self):
```

Reply:
It will be accessed by an external method and `mypy` or `ruff` may complain?

Reply:
Okey, will be applied in new commit